### PR TITLE
Freyja update to 1.4.4

### PR DESCRIPTION
**Pull request by PlushZ**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/freyja/freyja_aggregate_plot.xml [hunk 1]
```diff
@@ -71,13 +71,22 @@
                     </when>
                     <when value="no"/>
                 </conditional>
+                <param argument="--lineages" type="boolean" truevalue="--lineages" falsevalue=""
+                       checked="false" label="Use lineage specific breakdown"
+                       help="To modify the provide a lineage specific breakdown, the --lineages flag can be used"/>
```
Comment (bgruening) [inline tools/freyja/freyja_aggregate_plot.xml:76]: that helps is a bit useless in the Galaxy context

File: tools/freyja/freyja_aggregate_plot.xml [hunk 2]
```diff
@@ -71,13 +71,22 @@
                     </when>
                     <when value="no"/>
                 </conditional>
+                <param argument="--lineages" type="boolean" truevalue="--lineages" falsevalue=""
+                       checked="false" label="Use lineage specific breakdown"
+                       help="To modify the provide a lineage specific breakdown, the --lineages flag can be used"/>
+                <param argument="--mincov" type="float" min="0" max="100" value="60"
+                       label="Minimum genome coverage"
+                       help="By default, the minimum genome coverage is set at 60 percent. To adjust this, the --mincov option can be used (e.g. --mincov 75)."/>
```
Comment (bgruening) [inline tools/freyja/freyja_aggregate_plot.xml:79]: last part can be removed?

File: tools/freyja/freyja_aggregate_plot.xml [hunk 3]
```diff
@@ -92,6 +101,12 @@
                     <option value="MS" selected="true">month bins</option>
                     <option value="D">day bins</option>
                 </param>
+                <param argument="--lineages" type="boolean" truevalue="--lineages" falsevalue=""
```
Comment (bgruening) [inline tools/freyja/freyja_aggregate_plot.xml:104]: should those params be added to the macro? They seem to be used many times.

File: tools/freyja/macros.xml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <macros>
-    <token name="@TOOL_VERSION@">1.3.8</token>
-    <token name="@VERSION_SUFFIX@">0</token>
+    <token name="@TOOL_VERSION@">1.4.4</token>
+    <token name="@VERSION_SUFFIX@">1</token>
```
Comment (bgruening) [inline tools/freyja/macros.xml:4]: thise can be reset to 0

File: tools/freyja/freyja_demix.xml [hunk 5]
```diff
@@ -89,9 +90,9 @@ sed 's/$in_file/$sn/' abundances_raw.tsv > abundances.tsv
             <conditional name="sample_name">
                 <param name="source" value="auto"/>
             </conditional>
-            <output name="abundances" ftype="tabular">
+            <output name="abundances">
```
Comment (bgruening) [inline tools/freyja/freyja_demix.xml:93]: it that still a tabular output? If so you could add number of columns to the test

**Review by bgruening — APPROVED** [review]

File: tools/freyja/macros.xml [hunk 6]
```diff
@@ -129,6 +134,19 @@ freyja plot
...
+               help="Allows to use larger UShER barcodes library extended with GISAID non-public lineages"/>
+    </xml>
+    <xml name="plot_lineages">
+        <param argument="--lineages" type="boolean" truevalue="--lineages" falsevalue=""
+               checked="false" label="Use lineage specific breakdown"/>
+    </xml>
+    <xml name="plot_mincov">
+        <param argument="--mincov" type="float" min="0" max="100" value="60"
+               label="Minimum genome coverage"
+               help="By default, the minimum genome coverage is set at 60 percent."/>
```
Comment (wm75) [inline tools/freyja/macros.xml:149]: `60 percent` of what? Can you improve that help?

**Comment by pvanheus** [comment]
@PlushZ the help text (e.g. https://github.com/galaxyproject/tools-iuc/blob/7201c490f63bb7e85398d52c4d2e7cb37b6e90d9/tools/freyja/freyja_aggregate_plot.xml#L289) implies that the data format is separated by / whereas the Freyja website itself is a bit ambiguous, providing both a - separated and a / separated file in https://github.com/andersen-lab/Freyja/tree/main/freyja/data - has the format changed in recent versions?

**Comment by PlushZ** [comment]
> 



> @PlushZ the help text (e.g.
> 
> https://github.com/galaxyproject/tools-iuc/blob/7201c490f63bb7e85398d52c4d2e7cb37b6e90d9/tools/freyja/freyja_aggregate_plot.xml#L289
> 
> ) implies that the data format is separated by / whereas the Freyja website itself is a bit ambiguous, providing both a - separated and a / separated file in https://github.com/andersen-lab/Freyja/tree/main/freyja/data - has the format changed in recent versions?

No, in recent versions the format of metadata file has not changed, and I checked both formats on the latest version, it works on both. So I will change help section

**Comment by pvanheus** [comment]
> @PlushZ the help text (e.g.
> 
> https://github.com/galaxyproject/tools-iuc/blob/7201c490f63bb7e85398d52c4d2e7cb37b6e90d9/tools/freyja/freyja_aggregate_plot.xml#L289
> 
> ) implies that the data format is separated by / whereas the Freyja website itself is a bit ambiguous, providing both a - separated and a / separated file in https://github.com/andersen-lab/Freyja/tree/main/freyja/data - has the format changed in recent versions?

Replying to myself... I checked in with Joshua Levy and he stated that the date values are parsed using Pandas [to_datetime](https://pandas.pydata.org/docs/reference/api/pandas.to_datetime.html) so a variety of formats are acceptable. In the example documentation. perhaps ISO-8601-like YYYY-MM-DD should be used to avoid ambiguity about which field is day and which field is month?

**Pull request merged**